cvat/apps/organizations/views.py

```diff
@@ -67,7 +67,7 @@ class OrganizationViewSet(viewsets.GenericViewSet,
                    PartialUpdateModelMixin,
     ):
     queryset = Organization.objects.select_related('owner').all()
-    search_fields = ('name', 'owner')
+    search_fields = ('name', 'owner', 'slug')
```

Comment:
``name`` may be empty. It makes it impossible to find some organizations via search query